### ISSUE TITLE
[a11y]: [Dropdown ] A11y (Accessbility) violation

### ISSUE THREAD

**Issue by ivanybma** [carbon-design-system/carbon#19957]
### Package

@carbon/react

### Package version

1.82.1

### React version

18.2.0

### Automated testing tool and ruleset

IBM Accessibility Equal Access Toolkit: Accessibility Checker

### Description

Hi Carbon team,

As part of an IBM-wide requirement, a requirement is for DataStage UI (DataStage/ds-canvas) to be accessibility compliant, via the [IBM Equal Access Accessibility Checker tool]

<img width="1679" height="991" alt="Image" src="https://github.com/user-attachments/assets/ca36f3f6-c914-465f-ad54-f17bcfdf5e74" />

### Reproduction/example

@carbon/react

### Steps to reproduce

1. Create a ETL Flow using transformer node using https://www.ibm.com/products/datastage using chrome browser
For example: RowGen node -> Transformer Node -> Peek node
2. Add 1 column on Rowgen
3. Double Click on transformer node
4. If new transformer toggle button is not on, turn it on as it is highlighted below and open the find panel:

<img width="1952" height="1102" alt="Image" src="https://github.com/user-attachments/assets/7db46d04-6e4f-49bd-9342-46e9b981f274" />

5.Assuming Accessibility checker tool is installed Tool Link: Equal Access checker [https://github.com/IBMa/equal-access]
Run the accessibility checker against the new transformer with left side panel open. There should be 23 violations for accessibility.

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by ariellalgilmore** [comment]
Can you reproduce this in a [Stackblitz](https://stackblitz.com/github/carbon-design-system/carbon/tree/main/examples/vite?file=src%2Findex.scss)?

**Comment by heloiselui** [comment]
Hey, could you reproduce this in a [Stackblitz](https://stackblitz.com/github/carbon-design-system/carbon/tree/main/examples/vite?file=src%2Findex.scss)? really appreciate it!

**Comment by philljenkins** [comment]
@ivanybma Also, since this issue is "blocking" a product, "DataStage" (per description above) from FedRAMP compliance (although not explicitly in this issue), it's a higher priority to push this issue (and others) through the process.

**Comment by tay1orjones** [comment]
Hi @ivanybma! 👋  I can't reproduce this on the base Dropdown or the Dropdown with AILabel via the `decorator` prop, which it appears you might be using. The "main content" and "landmark element" violations shown in the screen recording below are just false-positives from the storybook environment. I'm not seeing any of the ones you've screenshotted.

https://github.com/user-attachments/assets/940c3611-2b91-42aa-ac7c-e4cd7e8560ad

The issues you're seeing are likely linked to your specific usage. We ask for a stackblitz reproduction link in the issue template because without it we can't see how you're authoring things. This makes it very difficult for us to help you identify what the problem could be.

I'm going to close this for now based on this and inactivity. If you're still encountering this issue, feel free to open a new issue or you can reopen this one with a reproduction link. Thanks!
